roipoly/roipoly.py

```diff
@@ -22,7 +22,7 @@ def deprecation(message):
 
 class RoiPoly:
 
-    def __init__(self, fig=None, color='b',
+    def __init__(self, fig=None, ax=None, color='b',
```

Comment:
could you add the new argument to the docstring as well?

Reply:
Just added.  The change slipped my notice.